content/Arduino Cloud/General Cloud/Install-the-Arduino-Cloud-Agent.md

```diff
@@ -29,21 +29,21 @@ You can now use [the Cloud Editor and Arduino Cloud](https://docs.arduino.cc/ard
 * Be sure that you have **administrator permission** on your machine; if the computer you are using is not your own personal computer, ask your IT administrator for an upgraded profile.
 * Sometimes the agent installation can be blocked by the firewall or antivirus software; in the latter case, you can try to **add the agent to your Antivirus whitelist**.
 
-### Old agent installation of the Arduino Create Agent found
+### Old agent installation of the Arduino Cloud Agent found
 
-When trying to launch the Arduino Create Agent on macOS, you may get this error:
+When trying to launch the Arduino Cloud Agent on macOS, you may get this error:
 
-`Old agent installation of the Arduino Create Agent found, please uninstall it before launching the new one`.
+`Old agent installation of the Arduino Cloud Agent found, please uninstall it before launching the new one`.
 
 The error can occur if an older installation is present. Follow these steps:
 
 1. Open Finder and go to your user folder. If you're having trouble finding it, right-click Downloads or Documents in the sidebar, and select _Show in Enclosing Folder_, or press <kbd>Shift</kbd> + <kbd>⌘</kbd> + <kbd>H</kbd>.
 
 2. Open the Applications folder (`~/Applications`) within your user folder.
 
-3. Manually remove the ArduinoCreateAgent folder, or run the `uninstall.app` inside it.
+3. Manually remove the ArduinoCloudAgent folder, or run the `uninstall.app` inside it.
```

Comment:
This particular folder will only be created by older versions of the software, that used the "Arduino Create Agent" name. For this reason, we want to keep the folder name as is.

```suggestion
3. Manually remove the ArduinoCreateAgent folder, or run the `uninstall.app` inside it.
```